src/__tests__/bootstrap.test.ts

```diff
@@ -44,6 +45,38 @@ test('bootstrap: error when required service not present', () => {
   ).toThrow('Missing services: decoder');
 });
 
+test('bootstrap: no error when required service is present', () => {
+  const core: EthqlPluginFactory = () => ({
+    name: 'core',
+    priority: 10,
+    serviceDefinitions: {
+      web3: {
+        implementation: {
+          factory: () => initWeb3({
+            jsonrpc: 'http://127.0.0.1:8545',
+            batching: false
+          }),
+        },
+      }
+    }
+  });
+
+  const plugin1: EthqlPluginFactory = () => ({
+    name: 'plugin1',
+    priority: 10,
```

Comment:
I suspect this is an ordering issue. Could you make the priority of `plugin1` 11, or add an `order` constraint such that `order.after` = `core`?

Reply:
@raulk Done. Still failing.